packages/frontend-shared/src/gql-components/topnav/VerticalBrowserListItems.vue

```diff
@@ -123,7 +121,12 @@ const props = withDefaults(defineProps <{
 })
 
 const browsers = computed(() => {
-  return (props.gql.browsers ?? []).slice().sort((a, b) => a.displayName > b.displayName ? 1 : -1)
+  const alphaSortedBrowser = (props.gql.browsers ?? []).slice().sort((a, b) => a.displayName > b.displayName ? 1 : -1)
+
+  // move the selected browser to the top to easily see selected browser version at the top when opening the dropdown
+  alphaSortedBrowser.some((browser, i) => browser.isSelected && alphaSortedBrowser.unshift(alphaSortedBrowser.splice(i, 1)[0]))
```

Comment:
might make this easier to read/debug with something like this to see whats selected? Not as optimized but we are dealing with fairly small arrays
```suggestion
  const [selectedBrowser] = _.remove(alphaSortedBrowser, (browser) => browser.isSelected)
  alphaSortedBrowser.unshift(selectedBrowser)
```